tabs/pid_tuning.js

```diff
@@ -521,11 +530,19 @@ TABS.pid_tuning.initialize = function (callback) {
                 var rcExpo = checkInput(rcExpoElement);
                 var rcExpoYaw = checkInput(rcExpoYawElement);
 
-                var maxAngularVelRoll = rateCurve.getMaxAngularVel(rateRoll, rcRate, rcExpo, useSuperExpo);
+                self.currentRates.rateRoll  = rateRoll * 100;
+                self.currentRates.ratePitch = ratePitch * 100;
+                self.currentRates.rateYaw   = rateYaw * 100;
+                self.currentRates.rcRate    = rcRate * 100;
+                self.currentRates.rcRateYaw = rcRateYaw * 100;
+                self.currentRates.rcExpo    = rcExpo * 100;
+                self.currentRates.rcExpoYaw = rcExpoYaw * 100;
```

Comment:
These will evaluate to `NaN` if range checking for the inputs fails. Better make them stay `undefined` and check for undefined before you are using them (or don't update them if the input is out of range).